porter/v1/cluster_control_plane.proto

```diff
@@ -295,6 +321,12 @@ service ClusterControlPlaneService {
   // ApplyPorterApp applies a porter app as defined by the provided porter.yaml file to a given deployment id
   rpc ApplyPorterApp(ApplyPorterAppRequest) returns (ApplyPorterAppResponse) {}
 
+  // DeletePorterApp deletes a porter app from a given project
```

Comment:
`DeletePorterApp deletes all instances of a porter app across deployment targets in a given project`, or something like that?